frontend/public/locales/en/apply-adult-child.json

```diff
@@ -631,9 +631,9 @@
     "no": "No"
   },
   "review-child-information": {
-    "read-carefully": "Please review this information carefully. The information provided will be used by Sun Life to create your child(ren)'s member account and will be reflected on their membership card.",
+    "read-carefully": "Please review this information carefully. The information provided will be used by Sun Life to create your child(ren)'s member account.",
     "page-title": "Review child(ren) information",
-    "page-sub-title": "{{child}} Information",
+    "page-sub-title": "{{child}} information",
```

Comment:
Would this make more sense as `{{child}}'s information`?
Same with `{{child}}'s access to dental insurance`.

Feel free to resolve this comment if you say otherwise.

Reply:
It matches Figma mockup. Testers will flag it otherwise, but fair point.

![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/114004123/7117132e-6564-46d9-8f74-0f9cc86a853f)

Reply:
Will keep inline with figma design for now